docs/resources/topic.md

```diff
@@ -51,7 +51,7 @@ resource "redpanda_network" "test" {
 
 resource "redpanda_cluster" "test" {
   name            = var.cluster_name
-  namespace_id    = redpanda_namespace.test.id
+  resourcegroup_id    = redpanda_resourcegroup.test.id
```

Comment:
Looks like this has to be linted

Reply:
Good catch, this is autogenerated from our examples. Just fixed in the example and now this should be fine. Thanks!